src/main/java/org/myrobotlab/framework/Service.java

```diff
@@ -1888,38 +1888,38 @@ public void subscribe(String topicName, String topicMethod, String callbackName,
       List<String> tnames = Runtime.getServiceNames(topicName);
       for (String serviceName : tnames) {
         MRLListener listener = new MRLListener(topicMethod, callbackName, callbackMethod);
-        send(Message.createMessage(getName(), serviceName, "addListener", listener));
+        send(Message.createMessage(getFullName(), serviceName, "addListener", listener));
       }
```

Comment:
getFullName() in subscriptions will break ServiceConfig.listeners - unless its fixed
That change is coming .. but there are some other filtering questions that need to be answered too

Reply:
Added a simple filter to `Service.getConfig()` that removes the local ID